src/model/CoilCoolingDXSingleSpeedThermalStorage.cpp

```diff
@@ -152,91 +162,13 @@ namespace model {
         }
       }
 
-      // AirLoopHVACUnitaryHeatCoolVAVChangeoverBypass
```

Comment:
So it can only be in an Unitary HVACComponent, right?

Reply:
Right.